coalib/settings/ConfigurationGathering.py

```diff
@@ -132,6 +133,20 @@ def warn_config_absent(sections, argument, log_printer):
     return False
 
 
+def get_default_section(section, sections):
+    default_section = ('.'.join(section.split('.')[:-1])
+                       if unescaped_search_for('.', section) else None)
```

Comment:
nah this is buggy, if I have several `.`s and one of them is escaped this may fail